src/features/scheduling/generation/hooks/useGenerateSectionSchedule..ts

```
@@ -0,0 +1,65 @@
+
+  if (!isSchedulingSection(section)) {
+    throw new Error("Cannot generate a schedule for a non-scheduling section.");
+  }
+
+  switch (section.type) {
+    case "BUNDLE":
+      const firstBundleOfSession = (await client.ensureInfiniteQueryData(getUseSectionListOptions(sessionId, {
+        where: [{ fieldPath: "type", operation: "==", value: "BUNDLE" }],
+        orderBy: [{ fieldPath: "startDate", direction: "asc" }],
+        limit: 1
+      }))).pages[0].docs[0] as Section;
```

Biome (2.5.5)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)